src/partials/about.html

```diff
@@ -0,0 +1 @@
+<div class="about">about</div>
```

Comment:
please change tag to section(or other semantic if needed) in all partials. And add id for using in future anchor links